functions/github/github.js

```diff
@@ -156,25 +156,29 @@ async function getLibraryMetadata(
     libraryChanges,
 ) {
   const libraryMetadata = {};
+  const allLibraryNames = [...libraryNames, ...Object.keys(libraryChanges)]
+      .filter((value, index, self) => self.indexOf(value) === index);
```

Comment:
I would rather abstract this into a separate extension method, to help readability:

```js
Object.assign(Array.prototype, {
  distinct() {
    return this.filter((value, index) => this.indexOf(value) === index);
  }
});
```

or a `removeDuplicates` function would also work.

Reply:
I moved the filtering to a function in utils, let me know if you think that's more readable. https://github.com/firebase/firebase-release-dashboard/pull/30/commits/c8b8ecb64c5c9ab5731d75b072f14aef9d51f846#diff-5bee11436cf44e5f9a9c7a540575a95539f361d48787e9e5fcabe1d1bcf5ee99R203

Reply:
Much better:)